### PR TITLE
Buttons on Additional form on validation and one-list price for Assoc…

### PR DESCRIPTION
**Pull request by EnitanOO**
…iated Services

[AB#13768](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13768)
[AB#13806](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13806)

[AB#9482](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9482)
[AB#13708](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/13708)

### CODE REVIEW

File: src/OrderFormAcceptanceTests.Steps/Steps/AdditionalServices.cs [hunk 1]
```diff
@@ -335,7 +335,7 @@ public void GivenTheSelectAdditionalServiceFormIsPresented()
             Test.Pages.OrderForm.ClickAddSolutionButton();
             Test.Pages.OrderForm.EditAdditionalServicesSectionDisplayed().Should().BeTrue();
         }
-
+        [StepDefinition(@"the User selects an Associated Service with only one list price")]
```
Comment (PhysicsJon) [inline src/OrderFormAcceptanceTests.Steps/Steps/AdditionalServices.cs:338]: There needs to be a blank line between the end of a method and the first step definition of the next method

**Review by PhysicsJon — CHANGES_REQUESTED** [review]
Just a single minor change required

File: src/OrderFormAcceptanceTests.Steps/Steps/AdditionalServices.cs (hunk 1 above)
Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Steps/Steps/AdditionalServices.cs:338]: sorting

Comment (EnitanOO) [reply src/OrderFormAcceptanceTests.Steps/Steps/AdditionalServices.cs:339]: Done

**Review by PhysicsJon — APPROVED** [review]

**Pull request merged**